src/components/layout/Header.js

```diff
@@ -1,28 +1,56 @@
-import React from "react";
-import { NavLink } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import { Link, NavLink, useLocation } from "react-router-dom";
 
 import classNames from "classnames";
+import useSize from "../../hooks/useSize";
 
 import "./Header.scss";
 
 import logo from "./logo.svg";
 
-const Header = ({ light, sticky }) => (
-  <header className={classNames({ light: light && !sticky, sticky })}>
-    <NavLink to="/" id="header--home-link">
-      <img src={logo} alt="" width="45" />
-      Team Health Check
-    </NavLink>
-    <NavLink activeClassName="header-nav--current" to="/interview/theory">
-      How to ?
-    </NavLink>
-    <NavLink activeClassName="header-nav--current" to="/interview/tryit">
-      Try it
-    </NavLink>
-    <NavLink activeClassName="header-nav--current" to="/visualize">
-      My team
-    </NavLink>
-  </header>
-);
+const Header = ({ light, sticky }) => {
+  const { medium } = useSize();
+  const { pathname } = useLocation();
+
+  const [hidden, setHidden] = useState(medium);
+
+  useEffect(() => {
+    setHidden(medium);
+  }, [medium]);
+
+  useEffect(() => {
+    setHidden(true);
+  }, [pathname]);
+
+  function toggleMenu() {
+    setHidden(!hidden);
+  }
+
+  return (
+    <header className={classNames({ medium, light: light && !sticky, sticky })}>
+      <span className="icon material-icons" onClick={toggleMenu}>
```

Comment:
This mecanism is neat but in responsive mode i'm not satisfied with a full width header. It would be nice to have the `burger menu` display just in a "thumbnail" or a bookmark, like (ascii style) :
```
----
Menu )
----
```

Reply:
Without the bottom border ?

Reply:
Like, float left ?